python/meterpreter/meterpreter.py

```diff
@@ -427,14 +437,15 @@ def crc16(data):
 @export
 def debug_print(msg):
     if DEBUGGING:
-        print(msg)
+        logging.debug(msg)
 
 @export
 def debug_traceback(msg=None):
     if DEBUGGING:
         if msg:
-            print(msg)
+            debug_print(msg)
         traceback.print_exc(file=sys.stderr)
```

Comment:
I think this should be deleted to avoid the double log, i.e. both to stderr via `print_exc` as well as the file and default logger